### PR TITLE
Fix: Redirect to Settings Screen on First App Launch

### PR DESCRIPTION
**Pull request by jylevergara**
## Summary

- On the first launch, the app now directs users to the Settings screen instead of the Home screen.
- Notification permission prompt appears on the Settings screen

### CODE REVIEW

File: app/(tabs)/index.tsx [hunk 1]
```diff
@@ -1,64 +1,336 @@
...
+  schedulePushNotification,
+  cancelScheduledNotification,
+  registerForPushNotificationsAsync,
+} from "@/hooks/notifications.hooks";
+import * as Notifications from "expo-notifications";
+import { NotificationTimeProps } from "@/helpers/props.helper";
 
-const phrases = data.phrases;
+// change this before pushing to dev
+const STORAGE_KEY = "test_schedule_preferences";
```
Comment (grailian) [inline app/(tabs)/index.tsx:25]: is it time to change this?

File: hooks/notifications.hooks.ts [hunk 2]
```diff
@@ -1,44 +1,85 @@
 import * as Notifications from "expo-notifications";
+import * as Device from 'expo-device';
+import Constants from 'expo-constants';
+import AsyncStorage from "@react-native-async-storage/async-storage";
```
Comment (grailian) [inline hooks/notifications.hooks.ts:4]: Still a mix of `'` vs `"` floating around.  Try installing prettier, and configuring VSCode to auto-format on save.

**Review by grailian — CHANGES_REQUESTED** [review]

File: app/(tabs)/index.tsx (hunk 1 above)
Comment (jylevergara) [reply app/(tabs)/index.tsx:25]: yes, sounds good

File: hooks/notifications.hooks.ts (hunk 2 above)
Comment (jylevergara) [reply hooks/notifications.hooks.ts:4]: I have a task for this and I will do it separate from this PR,  on the next update

**Pull request merged**